cardano-cli/cardano-cli.cabal

```diff
@@ -275,13 +275,18 @@ test-suite cardano-cli-golden
                         Test.Golden.Shelley.TextEnvelope.Certificates.OperationalCertificate
                         Test.Golden.Shelley.TextEnvelope.Certificates.StakeAddressCertificates
                         Test.Golden.Shelley.TextEnvelope.Certificates.StakePoolCertificates
+                        -- TODO intricate DELETE
                         Test.Golden.Shelley.TextEnvelope.Keys.ExtendedPaymentKeys
                         Test.Golden.Shelley.TextEnvelope.Keys.GenesisDelegateKeys
                         Test.Golden.Shelley.TextEnvelope.Keys.GenesisKeys
                         Test.Golden.Shelley.TextEnvelope.Keys.GenesisUTxOKeys
+                        -- TODO intricate DELETE
                         Test.Golden.Shelley.TextEnvelope.Keys.KESKeys
+                        -- TODO intricate DELETE
                         Test.Golden.Shelley.TextEnvelope.Keys.PaymentKeys
+                        -- TODO intricate DELETE
                         Test.Golden.Shelley.TextEnvelope.Keys.StakeKeys
+                        -- TODO intricate DELETE
```

Comment:
What's the reason for deleting these?

Reply:
This was so long ago that I honestly don't remember (however, I still think this PR is highly relevant). You can probably just remove these `TODO` comments.

Reply:
Thanks!